prac_05/emails.py

```diff
@@ -13,7 +13,7 @@ def main():
         username = get_name(email)
         enquiry = input(f"Is your name{username}? (Y/n)").upper()
         if enquiry != "Y" and enquiry != "":
-            real_name = input("Name :")
```

Comment:
The naming convention between real_name and username can be confusing
but it works for this exercise

Reply:
I will fix that 
Thanks for your feedback